### DISCUSSION TITLE
2일차 ros2 launch airobo_gazebo gazebo_world.launch.py 실행시 package 'gazebo_ros' not found 오류

### DISCUSSION THREAD

**Discussion by 04941** [Road-Balance/gcamp_ros2_basic#57] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89412598/179456267-08c91986-6c29-4a0a-9d1a-e86fb4bf16d2.png)
이 과정 시행 중 오류가 나서 실행이 안 됩니다.

오류 사진입니다.
![image](https://user-images.githubusercontent.com/89412598/179456092-fcba0350-c3f9-467b-adc3-f7d4ef1c4c8b.png)
![image](https://user-images.githubusercontent.com/89412598/179456623-c1f551b2-4199-4489-9c45-ed57e8faa603.png)
따라서  sudo apt install ros-foxy-gazebo_ros 시도해봤는데 설치가 안 되는 것 같습니다.
"package 'gazebo_ros' not found 오류 문구가 뜹니다.

**Comment by kimsooyoung** [comment, marked as answer]
```
ros-foxy-gazebo_ros  => ros-foxy-gazebo-ros 
```

위과 같이 수정해서 해보세요

**Reply by 04941** [reply]
감사합니다. 해결되었습니다!
